pkg/istio/rbac/common/util.go

```diff
@@ -429,8 +429,8 @@ type ServiceEnabled struct {
 }
 
 type ComponentEnabled struct {
-	serviceList   []ServiceEnabled
-	namespaceList []string
+	serviceMap    map[ServiceEnabled]string
```

Comment:
I think it will be easier to just make this `map[string]bool` instead of `map[ServiceEnabled]string`. That way the key will just be `namespace/service` instead of the entire struct object. It will also simplify the lookup as well since we don't need to create an object for lookup.